src/PrestaShopBundle/Resources/config/services/core/grid.yml

```diff
@@ -165,3 +165,10 @@ services:
     # Grid Query applicators
     prestashop.core.query.doctrine_search_criteria_applicator:
         class: 'PrestaShop\PrestaShop\Core\Grid\Query\DoctrineSearchCriteriaApplicator'
+
+    # Grid position updater
+    prestashop.core.grid.position.updater:
```

Comment:
```
- prestashop.core.grid.position.updater
+ prestashop.core.grid.position.doctrine_grid_position_updater
```

Damn! I'm tired to tell this again and again, we need an automated validator for that xD /c @eternoendless